project/services/csvRegistrationService.php

```diff
@@ -3,11 +3,30 @@
 include_once "../services/saveUserService.php";
 include_once "../security/randomPasswordGenerator.php";
 
+
+define('CSV_START_LINE', 1);
+define('CSV_LINE_DELIMITER', "\n");
+define('CSV_COLUMN_DELIMITER', ";");
+
+
 function processCsvFileAndSaveUsers__FAKE($csv) {
 
+    $results = array(
+        array (
+            "surname" => "PRIEZV",
+            "name" =>"MENO",
+            "email" => "EMAIL",
+            "school" => "SKOLA",
+            "schoolAddress" => "SKOLAAD",
+            "address" => "ADRESA",
```

Comment:
I think address cannot be single string, as we have in DB address but it is constructed from foreign keys to other tables, so we are no saving full string as address

Reply:
Maybe we can use what you described here? #67